### PR TITLE
Support import-mode

### LINKED ISSUE #231: Doesn’t respect `--import-mode`
You use `pytest`’s `import_path` function, but don’t pass the user’s preferred `--import-mode`.

### PR DESCRIPTION
**Pull request by flying-sheep**
Fixes #231

### CODE REVIEW

File: tests/test_doctestplus.py [hunk 1]
```diff
@@ -865,6 +865,28 @@ def test_doctest_subpackage_requires(subpackage_requires_testdir, caplog):
...
+            '''
+            >>> 1
+            1
+            '''
+    """)
+
+    a.join('testcode.py').write(pyfile)
+    b.join('testcode.py').write(pyfile)
+
+    mismatch = testdir.inline_run(str(testdir), "--doctest-plus", "--import-mode=prepend")
```
Comment (pllim) [inline tests/test_doctestplus.py:884]: What about testing for `append`?

**Comment by pllim** [comment]
Thanks! Please also add a change log entry to https://github.com/scientific-python/pytest-doctestplus/blob/main/CHANGES.rst

File: tests/test_doctestplus.py (hunk 1 above)
Comment (pllim) [reply tests/test_doctestplus.py:884]: Also this syntax appears to not work for some pytest versions. Maybe you can skip patching for pytest<6 unless you care about old pytest support?

**Review by pllim — APPROVED** [review]
LGTM, thanks! But I'll let Brigitta decide if we can use "Squash and Merge" or you have to do it manually.

**Comment by bsipocz** [comment]
changelog conflict, so I'm going ahead and do a squash while rebasing

**Review by bsipocz — APPROVED** [review]
Thanks @flying-sheep!

**Pull request merged**